src/storage/create.js

```diff
@@ -1,11 +1,13 @@
 const middy = require("middy");
 const { cors } = require("middy/middlewares");
+const uuid = require("uuid/v4");
 const { uploadDocument } = require("./documentService");
 
 const handleCreate = async event => {
   try {
-    const { document, ttl } = JSON.parse(event.body);
-    const receipt = await uploadDocument(document, Number(ttl));
+    const { document, id, ttl } = JSON.parse(event.body);
+    const docStoreId = id || uuid();
```

Comment:
should only accept an external uuid parameter if its already an existing queue number

Reply:
fixed.